packages/components/src/select-control/index.js

```diff
@@ -559,6 +563,7 @@ SelectControl.defaultProps = {
 	showClearButton: false,
 	hideBeforeSearch: false,
 	staticList: false,
+	autoComplete: 'off',
```

Comment:
Should this default to `new-password` since `off` doesn't work with Chrome?  Or do you think consumers should be responsible for implementing this?

Reply:
I think consumers should still be responsible for this, as `new-password` isn't a natural way of turning it off. And from what I read, Chrome could decide to use `off` again if they so choose (`off` works fine on Safari and Firefox).